src/coins.h

```diff
@@ -102,47 +102,25 @@ using CoinsCachePair = std::pair<const COutPoint, CCoinsCacheEntry>;
  * - unspent, FRESH, DIRTY (e.g. a new coin created in the cache)
  * - unspent, not FRESH, DIRTY (e.g. a coin changed in the cache during a reorg)
  * - unspent, not FRESH, not DIRTY (e.g. an unspent coin fetched from the parent cache)
- * - spent, FRESH, not DIRTY (e.g. a spent coin fetched from the parent cache)
  * - spent, not FRESH, DIRTY (e.g. a coin is spent and spentness needs to be flushed to the parent)
  */
 struct CCoinsCacheEntry
 {
 private:
     /**
      * These are used to create a doubly linked list of flagged entries.
-     * They are set in SetDirty, SetFresh, and unset in SetClean.
-     * A flagged entry is any entry that is either DIRTY, FRESH, or both.
+     * They are set in SetDirty and unset in SetClean.
+     * A flagged entry is any entry that is DIRTY or DIRTY and FRESH.
```

Comment:
```suggestion
     * A flagged entry is any entry that is DIRTY or DIRTY-and-FRESH.
```
(the or/and combination is confusing otherwise)

Reply:
Done.